server/src/main/java/edu/jhu/server/Board.java

```diff
@@ -65,6 +65,16 @@ public Board() {
       Weapon weapon = new Weapon(name);
     }
   }
+  
+  // Function needs to actually check if player's piece is in a room
+  public boolean isSuggestionValid(Player suggester) {
```

Comment:
i think this could be removed. Could use

    board.getLocationOf(player.suspect) instanceof Room

?